### ISSUE TITLE
Password and backup phrase not working

### ISSUE THREAD

**Issue by Soleone** [GetScatter/ScatterDesktop#265]
There was a [user today in Telegram](t.me/Scatter/73828) who swore he wrote down the password and backup phrase correctly - he was German so I believe he was meticulous 😂  - but apparently Scatter 10.1 refused to accept either of the two.

He also mentioned:

> I changed the pw 1 hour ago

We ended up having to suggest to reset Scatter and import the keys again. For some users like him that is very daunting, 5 private keys to be imported, reconnect Worbli, EOSForce and Telos. For someone more experienced no problem obviously.

There was another user yesterday or a few days ago who seemed to have the same problem (could have also been related to importing a Chrome backup, can't quite remember), I believe @nsjames mentioned the possibility of some data corruption during the upgrade, but not sure.

I just wanted to create this issue to log the problem at least and then we can collect potentially more reports here and try to find a pattern.

**Comment by LupusDei** [comment]
I have been experiencing the same issue.   Exported from chrome app -> imported to desktop app.  
Desktop app says "Welcome Back" and prompts for the passphrase.  I type the same one I used for the chrome app, same used to encrypt/decrypt the backup - but desktop app says it is no good.

**Comment by Soleone** [comment]
@LupusDei You might be running into a different issue, but not sure.

When you installed Scatter Desktop did it say "Welcome Back" right away? Or only after trying to import your backup?

If it is the former then that points to you having Scatter Desktop installed before, I know that happened to a few users that tried the old version of Scatter Desktop and even forgot about it. If that is the case then you could just reset Scatter first and then try importing your backup file from the Chrome extension again maybe.

**Comment by LupusDei** [comment]
Only showed "Welcome Back" after completing the backup.

Yea, might just setup manually.  Thanks.

**Comment by nsjames** [comment]
Hmm, I'm going to set up a script to just change my password N times and see if I can get the data to corrupt somewhere along the process. We see this every now and then and it's definitely not a fun problem for most users.

**Comment by nsjames** [comment]
Cleaning tickets for Scatter 11 testing release (makes this ticket no longer relevant)

**Comment by dapixcasey** [comment]
This is still an issue...

**Comment by nsjames** [comment]
@dapixcasey shouldn't be an issue on 11. There is a redundant backup file before disk and mnemonic phrases as password replacements no longer exist.

**Comment by dapixcasey** [comment]
@nsjames - The problem is resolved by pressing "d" at the login until the password reset button appears.

**Comment by living-life** [comment]
Please help! I'm having the same problem. 

I've logged into my schedule desktop for a long time and the password always worked. Now the password does not work and the backup phrase does not work. I'm on a MacBook Pro laptop.

Can you please help me get into the scatter account?

I prefer not to have to delete my account and re-import my private keys because I'm not savy programmer and I don't want to make a mistake in that step. 

Please help!

**Comment by DavidBueno77** [comment]
same problem here..  any help would be awsome.. I have backup password and phrasebut both dont work.. Tried a restore with json file but it says it has a problem decrypting..

**Comment by tanevanwifferen** [comment]
same here. can't get into my scatter anymore. please reopen, this is a serious issue

**Comment by DavidBueno77** [comment]
I fixed it in the end, but dont anymore know how. I remember getting help from Eos authority. Try to email one of those guys for help.. Try rohan@eosauthority.com.

**Comment by Jleist007** [comment]
happening for me too. says error decrypting backup. And now I see scatter charges money to open a eos wallet so im not doing that either.
